phys2bids/tests/test_bids_units.py

```diff
@@ -0,0 +1,12 @@
+from phys2bids.bids_units import bidsify_units
+
+
+def test_bidsify_units():
+    # test normal unit
+    bidsify_units("V")
+    # test unit with standard prefix
+    bidsify_units("centik")
+    # test unit with not standard prefix
+    bidsify_units("matV")
+    # test unit that's not bids standard
+    bidsify_units("mmlie")
```

Comment:
We should be checking that `bidsify_units( )` returns the correct units.

Reply:
true, completely forgot. I just checked the coverage

Reply:
done